src/components/DatePicker/DatePickerHeader/DatePickerHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { DropdownChevronDown, DropdownChevronUp } from "../../Icon/Icons";
+import moment from "moment";
+import { Moment } from "../types";
+import styles from "./DatePickerHeader.module.scss";
+import Icon from "../../Icon/Icon";
+
+interface DatePickerHeaderProps {
+  currentDate: Moment;
+  isMonthYearSelection: boolean;
+  onToggleMonthYearPicker: () => void;
+  hideNavigationKeys: boolean;
+  "data-testid"?: string;
+}
+
+const DatePickerHeader = (props: DatePickerHeaderProps) => {
+  const {
+    currentDate,
+    isMonthYearSelection,
+    onToggleMonthYearPicker,
+    hideNavigationKeys,
+    "data-testid": dateTestId
+  } = props;
+
+  const localedDated = moment(currentDate.valueOf());
+  const month = localedDated.format("MMMM");
+  const year = localedDated.format("YYYY");
+  const string = month + " " + year;
+  return (
+    <div className={styles.datePickerHeaderContainer}>
+      <div>{string}</div>
+      {!hideNavigationKeys && (
+        <button
+          data-testid={`${dateTestId}-year-toggle`}
+          type="button"
+          className={styles.button}
+          onClick={onToggleMonthYearPicker}
+        >
+          <div className={styles.buttonContent}>
+            <Icon
+              iconType={Icon?.type?.SVG}
+              // @ts-ignore
+              icon={isMonthYearSelection ? DropdownChevronUp : DropdownChevronDown}
+              iconSize={24}
+              clickable={false}
+              ignoreFocusStyle
```

Comment:
after using clickable false the icon should not be focusable and it should work without ignoreFocusStyle. It's not working?